lib/auth/github_authenticator.rb

```diff
@@ -6,6 +6,16 @@ def name
     "github"
   end
 
+  def enabled?
+    SiteSetting.enable_github_logins
+  end
+
+  def description_for_user(user)
+    info = GithubUserInfo.find_by(user_id: user.id)
+    return nil if info.nil?
```

Comment:
Are we able to return a blank string here instead? `info.blank? ? "" : (info.screen_name || "")`